features/features.json

```diff
@@ -1,4 +1,17 @@
  [
+  {
+    "title": "Direct image uploads in the forums",
+    "description": "Use assets.scratch.mit.edu as an image host on the forums easily by either clicking the upload image button, or CTRL+Y. Works on the forum posts and signatures. Unblocked for School Chromebook users!",
```

Comment:
Maybe a description like "Allows you to easily upload images to the Scratch forums for posts and signatures without having to use third party image uploading services."

Reply:
Just because we don't need to specify that it works for Chromebook users, and we also don't need to specify the image hosting platform.